data/autoyast_sle15/autoyast_containers_x86_64.xml

```diff
@@ -56,10 +56,51 @@
           <filesystem config:type="symbol">swap</filesystem>
           <mount>swap</mount>
         </partition>
+        
         <partition>
           <mountby config:type="symbol">device</mountby>
           <filesystem config:type="symbol">btrfs</filesystem>
           <mount>/</mount>
+          <subvolumes config:type="list">
```

Comment:
this profile is used by a lot of different tests , does it really make sense to have such setup for all of them  ? can we isolate this test in separate image ?

Reply:
That's a good point IMO. 
In general i would say that it makes sense. especially for testing purposes. and even more for btrfs as you have a dedicate partition for the containers. Now the question is it will break any other scenario for me somehow. i cant think any reason why it could be destructive but if you provide me with some links i can run some VRs.

Reply:
> In general i would say that it makes sense. especially for testing purposes. 

disagree . I always trying to avoid approach "test everything everywhere" . test should be **isolated** . in openQA test runs you can clearly see why . Imagine there is a bug in btrfs subvolumes  . in case this PR will be merged like this it will block **all** tests . In case you will **isolate** this into dedicated job you still have some test results

Reply:
> > In general i would say that it makes sense. especially for testing purposes.
> 
> disagree . I always trying to avoid approach "test everything everywhere" . test should be **isolated** . in openQA test runs you can clearly see why . Imagine there is a bug in btrfs subvolumes . in case this PR will be merged like this it will block **all** tests . In case you will **isolate** this into dedicated job you still have some test results

hm... I would interpret it as urgent bug and test blocker ( that would be finally the point to set blocker field in Bugzilla and if it cannot be soft-failed somehow ).

Reply:
Yeah sure, it is a blocker. But we have two possibilities still one is " we have a blocker bug all test failed we don't know anything else" or " we have blocker **in btrfs related test**, all test still working because they don't use subvolumes feature and what we found more". I just want to have second option and you?

Reply:
> Yeah sure, it is a blocker. But we have two possibilities still one is " we have a blocker bug all test failed we don't know anything else" or " we have blocker **in btrfs related test**, all test still working because they don't use subvolumes feature and what we found more". I just want to have second option and you?

We should definitely test other storage drivers than btrfs, hence we should have also container tests running on non-btrfs host.

Reply:
to be honest i am a bit lost with this. btrfs or anything else docker uses `/var` to store containers and meta. the xml just create dedicate partition for var. it has nothing to do with btrfs or subvolumes. 
In additional i have not found any job that will use the image and will be destructive. sorry i have missed something but it looks good to me.

Reply:
> to be honest i am a bit lost with this. btrfs or anything else docker uses `/var` to store containers and meta. the xml just create dedicate partition for var. it has nothing to do with btrfs or subvolumes.
> In additional i have not found any job that will use the image and will be destructive. sorry i have missed something but it looks good to me.

@jlausuch and I can send you output from `docker info`. Jose has assigned himself for testing openstack image which is xfs based image. 
HTH https://docs.docker.com/storage/storagedriver/select-storage-driver/